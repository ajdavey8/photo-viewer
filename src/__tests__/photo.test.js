import React from 'react';
import { shallow } from 'enzyme';
import renderer from 'react-test-renderer';
import Photo from '../components/photo';

describe(Photo, () => {
  const file = { "file": "1 Alan Berryhill 2018-04-17 person, outdoors, rain.jpg"};
  const component = shallow(<Photo image={file} />);

  it('renders and matches snapshot', () => {
    const tree = renderer
      .create(<Photo image={file} />)
      .toJSON();
    expect(tree).toMatchSnapshot();
  });

  it('contains the supplied image', () => {
    const image = component.find('img');
    expect(image.length).toEqual(1);
  });
});
