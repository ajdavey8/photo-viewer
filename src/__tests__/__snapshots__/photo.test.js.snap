// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`({ image }) =>
_react2.default.createElement('img', { src: require(\`../images/\${image.file}\`), alt: image.file, __source: { fileName: _jsxFileName, lineNumber: 4 }, __self: undefined }) renders and matches snapshot 1`] = `
<img
  alt="1 Alan Berryhill 2018-04-17 person, outdoors, rain.jpg"
  src="1 Alan Berryhill 2018-04-17 person, outdoors, rain.jpg"
/>
`;
